config/initializers/algoliasearch.rb

```diff
@@ -1,18 +1,4 @@
-def application_id
-  ENV.fetch('ALGOLIA_APP_ID')
-rescue KeyError => e
-  Rails.logger.error("Could not find environment variable. Defaulting to 'fake_algolia_app_id'. #{e.message}")
-  'fake_algolia_app_id'
-end
-
-def api_key
-  ENV.fetch('ALGOLIA_WRITE_API_KEY')
-rescue KeyError => e
-  Rails.logger.error("Could not find environment variable. Defaulting to 'fake_algolia_write_api_key'. #{e.message}")
-  'fake_algolia_write_api_key'
-end
-
 AlgoliaSearch.configuration = {
-  application_id: application_id,
-  api_key: api_key
+  application_id: ENV.fetch('ALGOLIA_APP_ID'),
```

Comment:
Might it instead be useful to only use the configuration when the variables are set? I don't think we don't need them to be there for any particular reason, do we?

Reply:
We shouldn't need them, but I suspect that the Vacancy model will throw a fit if we give it no config and then try to `include AlgoliaSearch`; and I'm uncomfortable with including fall-back variables in the config in case a production environment tries to use them, and we miss out on indexing some records.

I will try it.

Reply:
It's as I expected:

```
Algolia::AlgoliaError:
        API not initialized
```

fails about a third of the specs.